tests/conftest.py

```diff
@@ -250,18 +255,10 @@ def tmp_sqlite(tmpdir_factory):
     return os.path.join(fn.strpath, "test.db")
 
 
-@pytest.fixture(scope="function", autouse=True)
-def setup():
-    if Config.global_config.repository_type == "sql":
-        close_sql_database_session_connection()
-    init_config()
-    init_scheduler()
-    init_managers()
-    init_config()
```

Comment:
I would keep the setup. In case the previous test process is killed before the teardown method is executed

Reply:
In that case, we only need 1 fixture, with the "yield" command in the middle of all the init().

If I'm correct, the "setup" and "teardown" naming in pytest doesn't affect when the fixture is run.

Reply:
Something like
```python
@pytest.fixture(scope="function", autouse=True)
def clean_repository():
    if Config.global_config.repository_type == "sql":
        close_sql_database_session_connection()
    init_config()
    init_scheduler()
    init_managers()
    init_config()
    
    yield

    if Config.global_config.repository_type == "sql":
        close_sql_database_session_connection()
    init_config()
    init_scheduler()
    init_managers()
    init_config()
```